.github/workflows/create-jira-issues-for-community-activities.yml

```diff
@@ -60,6 +60,7 @@ jobs:
     timeout-minutes: ${{ inputs.timeout-minutes }}
     steps:
       - name: Create Jira issues for community activities
+        if: "!startsWith(github.head_ref, 'renovate')"
```

Comment:
In general, we need review issues to be created for Renovate PRs too.

Reply:
Okay, will delete this.